ios/versioned-react-native/ABI35_0_0/EXCalendar/ABI35_0_0EXCalendar/ABI35_0_0EXCalendarConverter.m

```diff
@@ -124,7 +124,7 @@ + (NSDictionary *)serializeSource:(EKSource *)source
   return @{
        @"id": source.sourceIdentifier,
        @"type": [ABI35_0_0EXCalendarConverter _sourceType:source.sourceType],
-       @"name": source.title
+       @"name": ABI35_0_0UMNullIfNil(source.title)
```

Comment:
We don't want to change versioned code in the client if you aren't going to update shell apps for standalone builds, it's better to still have that bug unfixed in older SDKs rather than have two different behaviors between the client and standalone. I'm fine if you will update shell apps for all these SDKs as well :D